kube/scripts/m3nsch_client.sh

```diff
@@ -34,6 +36,19 @@ fi
 
 AGENT_ENDPOINTS=$(get_endpoints)
 
+if [[ "$#" -eq 1 && "$1" == "start" ]]; then
+  (
+    IFS=","
```

Comment:
Is this being used? I'm confused

Reply:
Yup! This is some bash-foo that when we iterate over the comma-separated `$AGENT_ENDPOINTS` will treat `,` as the separator (it's usually ` `, `\t`, or `\n`). https://bash.cyberciti.biz/guide/$IFS